src/app/helpers/environmentVariablesHelper.js

```diff
@@ -98,6 +98,9 @@ let envVariables = {
   sunbird_azure_report_container_name: env.sunbird_azure_report_container_name || 'reports',
   sunbird_azure_account_name: env.sunbird_azure_account_name,
   sunbird_azure_account_key: env.sunbird_azure_account_key,
+  desktop_azure_crash_container_name: env.desktop_crash_container_name,
+  desktop_azure_account_name: env.desktop_azure_account_name,
```

Comment:
check with devops wheter we are going to reuse **sunbird_azure_account_name**, **sunbird_azure_account_key**